staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -201,7 +208,7 @@ func (c *celAdmissionController) Validate(
 		default:
 			deniedDecisions = append(deniedDecisions, policyDecisionWithMetadata{
 				PolicyDecision: PolicyDecision{
-					Action:  ActionDeny,
+					Action:  ActionAdmit,
```

Comment:
Should this still be ActionDeny?

Reply:
Thank you!

Reply:
Sorry messed that up flipping the private/public back and forth